app/src/main/java/com/duckduckgo/app/trackerdetection/blocklist/BlockListInterceptorApiPlugin.kt

```diff
@@ -58,7 +69,11 @@ class BlockListInterceptorApiPlugin constructor(
                 activeExperiment.isEnabled(CONTROL) -> config["controlUrl"]
                 else -> config["nextUrl"]
             } ?: return chain.proceed(request.build())
-            chain.proceed(request.url("$TDS_BASE_URL$path").build())
+            chain.proceed(request.url("$TDS_BASE_URL$path").build()).also { response ->
+                if (!response.isSuccessful) {
+                    pixel.fire(BLOCKLIST_TDS_FAILURE, mapOf("code" to response.code.toString()))
```

Comment:
Sending a pixel if the path on the remote config is wrong and the request fails.